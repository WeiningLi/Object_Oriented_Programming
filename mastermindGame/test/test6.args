121 20 1
