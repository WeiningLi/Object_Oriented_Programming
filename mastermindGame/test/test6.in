CBDFEDBACDBEFABDCEDD
Y
CBDFEDBACDBEFABDCEDD
Y
CBDFEDBACDBEFABDCEDD
Y
CBDFEDBACDBEFABDCEDD
Y
CBDFEDBACDBEFABDCEDD
Y
CBDFEDBACDBEFABDCEDD
Y
CBDFEDBACDBEFABDCEDD
Y
CBDFEDBACDBEFABDCEDD
N
