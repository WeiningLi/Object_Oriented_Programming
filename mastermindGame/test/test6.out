Welcome to Mastermind! Please enter your first guess:
1b,11w
You lost! The password was: AFCCCCEABACFDBCABFCA
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
2b,14w
You lost! The password was: DABDCEAACFABEEAEDDFB
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
2b,11w
You lost! The password was: FFCAFECAFEEEEBCAFDAB
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
2b,14w
You lost! The password was: CACDCEBFEFDCDDABFAAB
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
6b,9w
You lost! The password was: ABFBABAEFDBACADCFEDB
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
1b,16w
You lost! The password was: ADEDDBAFEBDDAEDFBECE
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
4b,8w
You lost! The password was: ABCABFCAFCBFBCEFCFAB
Would you like to play again? (Y/N): 
Welcome to Mastermind! Please enter your first guess:
3b,12w
You lost! The password was: CDBFFCACDAABBBDABEBB
Would you like to play again? (Y/N): 
